Lil-Peep/src/styles/modules/featured-kits.scss

```diff
@@ -0,0 +1,387 @@
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  justify-content: flex-end;
+  text-align: center;
+
+  .big-banner-title {
+    font-size: 20px;
+    margin-bottom: 0;
+  }
+
+  .big-banner-description {
+    font-family: 'Lato', sans-serif;
+    font-size: 12px;
+    margin: 0;
+  }
+
+  .view-in-detail {
+    background-color: #4a90e2;
+    border-radius: 3px;;
+    border-style: none;
+    margin: 10px;
+    padding: 10px 0;
+    text-align: center;
+
+    a {
+      text-decoration: none;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ed7f03;
+      }
+    }
+
+    .view-detail-button {
+      color: #fff;
+      font-family: 'Oswald', sans-serif;
+      font-weight: bold;
+      margin: 0;
+    }
+  }
+}
+
+.small-banner {
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  box-shadow: inset 0 0 0 1000px rgba(245, 166, 35, 0.8);
+  color: #FFFFFF;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 20px;
+
+  .small-banner-tittle {
+    font-size: 20px;
+    margin: 0;
+
+    .small-banner-description {
+      font-size: 40px;
+      font-weight: bold;
+      margin: 0;
+    }
+  }
+}
+
+.mid-banner-content {
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: flex-end;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+
+  .mid-banner-saylor {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .mid-banner-archeologist {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+ button {
+    background-color: Transparent;
+    border-radius: 2px;
+    border-style: solid;
+    font-family: 'Lato', sans-serif;
+    margin-bottom: 12px;
+    margin-left: 5px;
+    margin-right: 5px;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  .button-sailor {
+    border-color: #627387;
+
+    .h1-sailor {
+      color: #627387;
+      margin: 0;
+      padding: 10px ;
+    }
+  }
+  .button-archeologist {
+    border-color: #FFFFFF;
+
+    .h1-archeologist{
+      color: #FFFFFF;
+      margin: 0;
+      padding: 10px 0;
+    }
+  }
+}
+
+@media only screen and (width: 768px) {
+.big-banner {
+  background-position: center;
+  background-size: cover;
+
+  .big-banner-description {
+     margin-bottom: 20px;
+   }
+
+   .small-banner {
+     background-position: bottom;
+     background-size: cover;
+   }
+
+   .mid-banner-saylor {
+     background-position: center;
+     background-size: cover;
+   }
+
+   .mid-banner-archeologist {
+     background-position: top;
+   }
+ }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1279px) {
+.flex-grid {
+  display: flex;
+  margin-bottom: 20px;
+
+  .big-banner {
+    background-position: left top;
+    background-size: cover;
+    display: flex;
+    flex: 2;
+    height: 400px;
```

Comment:
This could be a variable defined in `variables.scss`, something like `$big-banner-height`, perhaps.